config.sh

```diff
@@ -38,7 +38,7 @@ LISK_CONFIG=""
 
 LISK_SCRIPTS_DIR="lisk-scripts-0.2.0"
 LISK_SCRIPTS_FILE="$LISK_SCRIPTS_DIR.tar.gz"
-LISK_SCRIPTS_URL="https://github.com/LiskHQ/lisk-scripts/archive/0.2.0.tar.gz"
+LISK_SCRIPTS_URL="https://github.com/LiskHQ/lisk-scripts/archive/v0.2.0.tar.gz"
```

Comment:
Latest version is v0.2.1.

If we're able to keep the 0.2.0 series compatible between 1.1 and 1.0, we don't need a 0.3.0 series. Otherwise, we'd want a 0.3.0 version and have v0.2.1 in the 1.0.0 branch